src/skfolio/preprocessing/_returns.py

```diff
@@ -14,6 +14,7 @@ def prices_to_returns(
     log_returns: bool = False,
     nan_threshold: float = 1,
     join: str = "outer",
+    index_intersect: bool = True,
```

Comment:
I was thinking about `drop_inceptions_nan` to be very explicit

Reply:
Done